conwayste/src/ui/pane.rs

```diff
@@ -97,6 +104,50 @@ impl Pane {
         }
     }
     */
+
+    /// Adds the vector of children to the parent, and shrink the parent to fit the widgets with
+    /// padding on all sides. The widgets will have the padding added to their x and y coordinates.
+    /// Therefore it is suggested that the upper left widget in `children` be at `(0, 0)`.
+    ///
+    /// # Errors
+    ///
+    /// It is an error if the parent does not have an `add` method for adding `Widget`s.
```

Comment:
When would this occur?

Reply:
Never. It's left over from before this was a method of `Pane`. It just took a `Box<dyn Widget>` before. Good catch!